packages/nx-payload/src/plugins/utils/write-targets-to-cache.ts

```diff
@@ -0,0 +1,6 @@
+import { type TargetConfiguration, writeJsonFile } from '@nx/devkit';
+
+export const writeTargetsToCache = (
+  cachePath: string,
+  targets: Record<string, Record<string, TargetConfiguration>>
+) => writeJsonFile(cachePath, targets);
```

Comment:
It will be handled by Nx process
```suggestion
export const writeTargetsToCache = (
  cachePath: string,
  targets: Record<string, Record<string, TargetConfiguration>>
) => writeJsonFile(cachePath, targets);
```